docs/src/components/home/ProductsSwitcher.tsx

```diff
@@ -92,7 +92,7 @@ const ProductsSwitcher = ({
       icon={<IconImage name="product-advanced" />}
       name={
         <Box component="span" display="inline-flex" alignItems="center">
-          Advanced&nbsp;
+          X Pro&nbsp;
```

Comment:
In this context, it feels that targeting the paid version of MUI X is too limiting. I would propose we don't mention "Pro".

Reply:
I personally think that "X Advanced" when "X" only feels too lonely is a great tradeoff.

Reply:
The downside of using _Advanced_ is that when referring to the paid version of the advanced components, we'd then have _X Advanced Pro_, which sounds a bit too ridiculous 😅 We'd be using half of the paid products words out there:  _X Advanced Pro Premium Black..._ 😬 

Feeling like the best way to go is MUI Core and MUI X?!

Reply:
Just pushed a commit with changing _**Core**_ → _**MUI Core**_ and _**X Pro**_ → _**MUI X**_.

Reply:
I'm resolving this since no one raised any concerns!